gnucash/import-export/qif-imp/qif-dialog-utils.scm

```diff
@@ -747,21 +747,9 @@
           (pref-match
            (cadr pref-match))
 
-          ;; Guess NYSE for symbols of 1-3 characters.
-          ((< l 4)
-           GNC_COMMODITY_NS_NYSE)
-
-          ;; Guess NYSE for symbols of 1-3 characters
-          ;; followed by a dot and 1-2 characters.
-          ((and d
-                (< l 7)
-                (< 0 d 4)
-                (<= 2 (- l d) 3))
-           GNC_COMMODITY_NS_NYSE)
-
-          ;; Guess NASDAQ for symbols of 4 characters.
-          ((= l 4)
-           GNC_COMMODITY_NS_NASDAQ)
+          ;; Guess SHARE for symbols of 1-4 characters.
+          ((<= l 4)
+           GNC_COMMODITY_NS_SHARE)
```

Comment:
I  suspect this won't work as GNC_COMMODITY_NS_SHARE is not defined in scheme code. GNC-COMMODITY-NS-SHARE probably is (thanks to the wrapper for gnc-commodity.h), so you can try to use that instead.

Reply:
Changing all GNC_COMMODITY_NS_ to GNC-COMMODITY-NS- results in 
qif-dialog-utils.scm:739:51: warning: possibly unbound variable `GNC-COMMODITY-NS-MUTUAL'